open-metadata-implementation/access-services/cognos/cognos-server/src/main/java/org/odpi/openmetadata/accessservices/cognos/utils/EntityPropertiesUtils.java

```diff
@@ -0,0 +1,108 @@
+/* SPDX-License-Identifier: Apache-2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.accessservices.cognos.utils;
+
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstanceProperties;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.InstancePropertyValue;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.MapPropertyValue;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.instances.PrimitivePropertyValue;
+import org.odpi.openmetadata.repositoryservices.connectors.stores.metadatacollectionstore.properties.typedefs.PrimitiveDefCategory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Map;
+
+public class EntityPropertiesUtils {
+
+    private static final Logger log = LoggerFactory.getLogger(EntityPropertiesUtils.class);
+
+
+    public static PrimitivePropertyValue createPrimitiveStringPropertyValue(String value) {
```

Comment:
Simialrly see common services and repository helper

Reply:
replaced with calls to OMRSRepositoryPropertiesUtilities in the class
EntityPropertiesBuilder https://github.com/odpi/egeria/pull/3044/commits/39c84a6c031f3af741e34a79ef77bab68549f5cb